### PR TITLE
Support sdt info

### PR DESCRIPTION
**Pull request by P-Cao**
- Add support for sdt info
- Add UT

# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [ ] I have performed Ansible Sanity test using --docker default
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by sachin-apa — APPROVED** [review]

File: plugins/modules/info.py [hunk 1]
```diff
@@ -1902,6 +1913,170 @@
         "jobId": "Job-10d75a23-d801-4fdb-a2d0-7f6389ab75cf",
         "rcmapproved": false
     }]
+SDT:
```
Comment (trisha-dell) [inline plugins/modules/info.py:1916]: please arrange the return values in alphabetic order.

Comment (trisha-dell) [reply plugins/modules/info.py:1916]: Rename the return value from SDT to sdt

File: plugins/modules/info.py [hunk 2]
```diff
@@ -1902,6 +1913,170 @@
...
+                            type: str
+    sample: [{
+        "authenticationError": "None",
+        "certificateInfo": null,
+        "discoveryPort": 8009,
+        "faultSetId": null,
+        "id": "8bddf18b00000000",
+        "ipList": [
+            {
+                "ip": "172.171.3.21",
```
Comment (trisha-dell) [inline plugins/modules/info.py:2025]: Please mask the IPs. Applicable for all instances.

**Comment by sachin-apa** [comment]
run e2e

File: plugins/modules/info.py (hunk 1 above)
Comment (P-Cao) [reply plugins/modules/info.py]: Updated

File: plugins/modules/info.py (hunk 2 above)
Comment (P-Cao) [reply plugins/modules/info.py]: Updated

**Review by sachin-apa — APPROVED** [review]

**Review by Bhavneet-Sharma — APPROVED** [review]

**Comment by P-Cao** [comment]
SonarQube
image

CheckMarx:
image

**Review by trisha-dell — APPROVED** [review]

**Pull request merged**